src/client/js/dialog.js

```diff
@@ -54,7 +56,11 @@ async function openDialog (path) {
       }
     })
 
-    if (!res.ok) throw new Error('Bad fetch response')
+    if (!res.ok) {
+      throw new ClientError('Something went wrong. Please try again or come back later.', {
+        action: 'toast'
```

Comment:
nit: We define an enum in src/client/js/error.js:
https://github.com/mozilla/blurts-server/blob/a80f61a40fd6338ebcd3392fe5de52cec3c8365a/src/client/js/error.js#L6-L9

Not sure if we should be exporting that, importing it here, and using `ErrorActionTypes.Toast`.

Reply:
Done in https://github.com/mozilla/blurts-server/pull/3010/commits/18a61ed85714578ca0a05edf669d2bdb2080828c.